grpc/client/src/main/java/io/helidon/grpc/client/GrpcChannelDescriptor.java

```diff
@@ -208,6 +215,7 @@ public Builder sslDescriptor(GrpcTlsDescriptor tlsDescriptor) {
          *
          * @see io.grpc.ManagedChannelBuilder#defaultLoadBalancingPolicy(String)
          */
+        // @ConfiguredOption
```

Comment:
This seems like not needed here.

Reply:
Yes, the code does not seem to use this hence why I commented it out, but forgot to remove it. Removing this.